config/backend-config/backend-config.go

```diff
@@ -227,6 +227,9 @@ func MakePostRequest(url string, endpoint string, data interface{}) (response []
 	}
 
 	body, err := IoUtil.ReadAll(resp.Body)
+	if err != nil {
+		pkgLogger.Errorf("ConfigBackend: Error parsing response body, Error: %s", err.Error())
```

Comment:
Use %v instead and give err as an argument instead of err.Error(). That way wrapped errors will also get logged

Reply:
done